pallets/confidential-asset/src/lib.rs

```diff
@@ -76,48 +71,35 @@ pub struct MercatAccount {
     pub encryption_pub_key: EncryptionPubKey,
 }
 
+type Identity<T> = identity::Module<T>;
+
 decl_storage! {
     trait Store for Module<T: Trait> as ConfidentialAsset {
 
         /// Contains the mercat accounts for an identity.
         pub MercatAccounts get(fn mercat_account):
-            double_map hasher(twox_64_concat) IdentityId,
-            hasher(blake2_128_concat) EncryptedAssetIdWrapper
-            => MercatAccount;
+              double_map hasher(twox_64_concat) IdentityId,
+              hasher(blake2_128_concat) EncryptedAssetIdWrapper
+              => MercatAccount;
 
         /// Contains the encrypted balance of a mercat account.
         pub MercatAccountBalance get(fn mercat_account_balance):
-            double_map hasher(twox_64_concat) IdentityId,
-            hasher(blake2_128_concat) EncryptedAssetIdWrapper
-            => EncryptedBalanceWrapper;
-
-        /// Contains the list of all valid ticker names.
-        /// The process around creating and storing this data will likely change as a result of
-        /// CRYP-153.
-        pub ValidAssetIds get(fn valid_asset_ids): Vec<AssetId>;
+              double_map hasher(twox_64_concat) IdentityId,
+              hasher(blake2_128_concat) EncryptedAssetIdWrapper
+              => EncryptedBalanceWrapper;
+
+        /// List of Tickers of the type ConfidentialAsset.
+        /// () -> List of confidential tickers
```

Comment:
```suggestion
```
Doesn't really add anything.

Reply:
Also, storage migration?

Reply:
Other storage API in other pallets are documented like this, e.g.:
```
        /// Ticker registration details.
        /// (ticker) -> TickerRegistration
        pub Tickers get(fn ticker_registration): map hasher(blake2_128_concat) Ticker => TickerRegistration<T::Moment>;
```
by `() -> List of confidential tickers` I mean this getter doesn't take an input, and returns a list of assets.

what do you mean by storage migration?

Reply:
- Those storage items are typically maps, where details like the hasher, etc. obscure semantics, but this is just a vector, so I still don't think the comment adds anything.

- Storage migration refers, in this case, to writing code that renames the old storage item into the new one, by reading the storage item, removing the old one, and then putting it under the new storage item name.